tests/integration/client/sdk/v1/test_records.py

```diff
@@ -57,6 +57,50 @@ def test_dataset():
 
 @pytest.mark.asyncio
 class TestRecordsSDK:
+    @pytest.mark.parametrize("role", [UserRole.owner, UserRole.admin])
+    async def test_update_record_with_vectors(self, owner: User, role: UserRole) -> None:
```

Comment:
```suggestion
    def test_update_record_with_vectors(self, owner: User, role: UserRole) -> None:
```

Reply:
I remove the use of the `UserFactory` to apply your changes.